prow/config.yaml

```diff
@@ -5211,11 +5212,11 @@ periodics:
         secretName: ssh-key-secret
 
 - interval: 1h
-  name: ci-kubernetes-e2e-gce-new-master-gci-kubectl-skew
+  name: ci-kubernetes-e2e-gce-new-master-gci-kubectl-skew-serial
   spec:
     containers:
     - args:
-      - --timeout=140
+      - --timeout=90
```

Comment:
this was what the timeout is complaining :-) keep it 140, or reduce the one from config.json